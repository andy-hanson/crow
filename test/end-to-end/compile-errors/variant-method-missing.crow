main void()
	info log x.foo
	info log y.foo
	()

v variant
	foo string()
x record v variant-member
foo string(_ x)
	"x"
y record v variant-member
