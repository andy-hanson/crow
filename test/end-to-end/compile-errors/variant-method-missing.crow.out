exit code: 1
stdout:
info: x
stderr:
test/end-to-end/compile-errors/variant-method-missing.crow 11:10-11:26 Could not find a function 'foo string(_ y)'.
Uncaught exception: Reached compile error
	at foo
	at main
